src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -103,6 +102,10 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
 
     @Input
     @Optional
+    String[] networkAliases
+
+    @Input
+    @Optional
```

Comment:
I added the networkAliases property to the task that can be set to assign aliases to the container.
Other containers in the network will then be able to reach it using that alias, e.g. `http://some-alias/`.